docs/source/model_design/vivarium_model_components/risk_factors/risk_effects/index.rst

```diff
@@ -126,9 +175,14 @@ The mathematical expressions are mainly fall into two categories:
      - :math:`PAF = \frac{E(RR_e)-1}{E(RR_e)}`
      - :math:`E(RR_e) = p \times RR + (1-p)`
  - risk exposure is continuous distributed:
-     - :math:`i = i \times (1-PAF) \times rr^{max(e−tmrel,0)/scalar}`
-     - :math:`PAF = \frac{E(RR_e)-1}{E(RR_e)}`
-     - :math:`E(RR_e) = \int_{lower}^{upper}rr^{max(e−tmrel,0)/scalar}p(e)de`
+     - risk effect has a log-linear "dose-response" relationship with exposure:
+         - :math:`i = i \times (1-PAF) \times rr^{max(e-tmrel,0)/scalar}`
```

Comment:
This line existed previously, so perhaps this is out of scope for this PR, but the notation here is confusing because the variable $i$ is being used for two different things: On the LHS, $i$ is the individual incidence hazard for a single simulant, whereas on the RHS, $i$ is the population-level incidence rate.